doc/source/installation.rst

```diff
@@ -1,79 +1,94 @@
+contains Python bindings to the `ComPWA backend
+<https://github.com/ComPWA/ComPWA>`_, you first need to install certain C++
+prerequisites.
 
-.. code-block:: shell
+What you definitely need to install, is:
 
-  conda config --add channels conda-forge
-  conda config --set channel_priority strict
+* a build tool: `cmake <https://cmake.org/>`_
+* a C++ compiler:
+  `gcc <https://gcc.gnu.org/>`_ or `clang <https://clang.llvm.org/>`_
+* `Boost <https://www.boost.org/>`_
+* `Python3 <https://www.python.org/downloads/>`_
 
-Now, create a new environment with `all required dependencies
-<https://github.com/ComPWA/pycompwa/blob/master/requirements.txt>`_
-installed:
+The following package are optional:
 
-.. code-block:: shell
+* `ROOT <https://root.cern.ch/downloading-root>`_ and/or `Minuit2
+  <http://seal.web.cern.ch/seal/snapshot/work-packages/mathlibs/minuit/>`_
+* `GSL <https://www.gnu.org/software/gsl/>`_
+* `Geneva <https://www.gemfony.eu/>`_
+
+It is highly recommended to install ROOT `with Minuit2 enabled
+<https://root.cern.ch/building-root>`_. Without it, ComPWA will have only
+limited functionality.
+
+.. hint::
 
-  conda create -n compwa --file requirements.txt
+  If you have a Linux machine with `apt <https://wiki.debian.org/Apt>`_ and
+  with administrator rights, you can run the following:
 
-Here, we call the environment ``compwa``, but you can give it any name. Now, go
-into this environment with ``conda activate compwa``, so we can install
-pycompwa there:
+  .. code-block:: shell
+
+    sudo apt update -y
+    sudo apt install -y cmake gcc git libboost-all-dev python3
+
+  ROOT with Minuit2 can be most easily installed by `downloading a suitable
+  binary for your machine <https://root.cern.ch/downloading-root>`_.
+
+
+Installation through pip
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once you have these dependencies installed, you can install `pycompwa` through
+`pip <https://pypi.org/project/pip/>`_. You also need to install `scikit-build
+<https://scikit-build.readthedocs.io/en/latest/>`_, because it is used as a
+build tool for the ComPWA backend:
 
 .. code-block:: shell
 
   pip install scikit-build
   pip install pycompwa
 
-Note that ``scikit-build`` has to be installed first.
+Et voilà, that's it! You can try out whether the installation works by running:
 
-That's it! You can now go through the :doc:`usage/workflow` to learn how to use
-:mod:`pycompwa`.
+.. code-block:: python
 
-.. tip::
+  import pycompwa
 
-    Of course, pycompwa can also be used with `jupyter
-    <https://jupyter.org/>`_. You can install jupyter in your virtual Conda
-    environment via ``conda install jupyter``. Then, just navigate to the
-    `examples <https://github.com/ComPWA/pycompwa/tree/master/examples>`_
-    folder and run ``jupyter notebook``.
+from the Python interpreter. If that works, you can try out some of the
+examples from the :doc:`usage` page.
+
+Note that you can :command:`pip` **only allows you to install specific
```

Comment:
delete: you can

Reply:
https://github.com/ComPWA/pycompwa/pull/83/commits/c2994feaa9ecafa72674a1163f7f8a4a45f0f805